packages/migration/ecosysystem.go

```diff
@@ -650,13 +650,14 @@ var (
 		"amount" decimal(30) NOT NULL DEFAULT '0',
 		"comment" text NOT NULL DEFAULT '',
 		"block_id" int  NOT NULL DEFAULT '0',
-		"txhash" bytea  NOT NULL DEFAULT ''
+		"txhash" bytea  NOT NULL DEFAULT '',
+		"created_at" timestamp DEFAULT NOW()
 		);
 		ALTER TABLE ONLY "%[1]d_history" ADD CONSTRAINT "%[1]d_history_pkey" PRIMARY KEY (id);
 		CREATE INDEX "%[1]d_history_index_sender" ON "%[1]d_history" (sender_id);
 		CREATE INDEX "%[1]d_history_index_recipient" ON "%[1]d_history" (recipient_id);
 		CREATE INDEX "%[1]d_history_index_block" ON "%[1]d_history" (block_id, txhash);
-		
+		CREATE INDEX "%[1]d_history_index_created" ON "%[1]d_history" (created_at);
```

Comment:
Doesn't sure that that index is good idea. Because of - yes, it will work if amount of records fetched by index is small, but it probably be large, because money transfer history is a large table. So, query planner will switch to FULL SCAN, making this index obsolete